tests/qa_automation/kernel_lvm2.pm

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Add 4 more kernel regression testsuits
+# Maintainer: Nathan Zhao <jtzhao@suse.com>
+
+use base "qa_run";
+use strict;
+use testapi;
```

Comment:
why you need testapi here ?

Reply:
I forget. This module is derived from kernel_lvm2_120.pm, which was written a long time ago.

Reply:
yeah it's probably not needed but other test modules do the same. All qa_automation modules should be handled the same so you could remove it everywhere in another PR